pkg/apiserver/conprof/service.go

```diff
@@ -81,7 +81,7 @@ func newService(lc fx.Lifecycle, p ServiceParams) *Service {
 func registerRouter(r *gin.RouterGroup, auth *user.AuthService, s *Service) {
 	endpoint := r.Group("/continuous_profiling")
 
-	endpoint.Use(utils.MWForbidByFeatureSupport(IsFeatureSupport(s.params.Config)))
+	endpoint.Use(utils.MWForbidByFeatureSupport(FeatureFlagConprof.IsSupport(s.params.Config.FeatureVersion)))
```

Comment:
This is too long.

Reply:
How about integrating gin middleware into `FeatureFlag` struct.

Reply:
It's a little strange.

Reply:
Updated. It's still long, but the purpose is much clearer.